analyses/survival-analysis/survival-analysis_histology.Rmd

```diff
@@ -83,19 +87,16 @@ tp53_scores <- readr::read_tsv(tp53_file) %>%
 
 # read in telomerase scores
 tel_scores <- readr::read_tsv(tel_stranded_file) %>%
-  dplyr::mutate(Kids_First_Biospecimen_ID_RNA = SampleID,
-        telomerase_score = NormEXTENDScores) %>%
-  dplyr::select(Kids_First_Biospecimen_ID_RNA, telomerase_score) %>%
+  dplyr::rename(Kids_First_Biospecimen_ID_RNA = SampleID,
+                telomerase_score = NormEXTENDScores) %>%
   dplyr::arrange(Kids_First_Biospecimen_ID_RNA)
 
 # combine tp53 scores and telomerase score to the dataframe
-meta_indep <- tp53_scores %>%
-  left_join(tel_scores) %>%
-  left_join(histologies) %>%
-  # remove samples without RNA-Seq since both classifiers' results use RNA-Seq
-  dplyr::filter(!is.na(Kids_First_Biospecimen_ID_RNA)) %>%
+meta_indep <- histologies_rna %>%
+  left_join(tp53_scores) %>%
+  left_join(tel_scores) %>% 
```

Comment:
Adding an explicit `by` here might be helpful for folks reading the code.

Reply:
Thanks - added `by = "Kids_First_Biospecimen_ID_RNA"` in the newest push.